libs/payments/ui/src/lib/client/components/CheckoutForm.tsx

```diff
@@ -135,71 +146,101 @@ export function CheckoutForm({ readOnly, cart, locale }: CheckoutFormProps) {
   const nonStripeFieldsComplete = !!fullName;
 
   return (
-    <Form.Root className="flex flex-col gap-4" onSubmit={submitHandler}>
-      {!isPaymentElementLoading && (
-        <Form.Field name="name" serverInvalid={hasFullNameError}>
-          <Form.Label className="font-medium text-sm text-grey-400 block mb-1 text-start">
-            <Localized id="payment-name-label">
-              Name as it appears on your card
-            </Localized>
-          </Form.Label>
-          <Form.Control asChild>
-            <input
-              className="w-full border rounded-md border-black/30 p-3 placeholder:text-grey-500 placeholder:font-normal focus:border focus:!border-black/30 focus:!shadow-[0_0_0_3px_rgba(10,132,255,0.3)] focus-visible:outline-none data-[invalid=true]:border-alert-red data-[invalid=true]:text-alert-red data-[invalid=true]:shadow-inputError"
-              type="text"
-              data-testid="name"
-              placeholder={l10n.getString(
-                'payment-name-placeholder',
-                {},
-                'Full Name'
-              )}
-              readOnly={readOnly}
-              value={fullName}
-              onChange={(e) => {
-                setFullName(e.target.value);
-                setHasFullNameError(!e.target.value);
-              }}
-            />
-          </Form.Control>
-          {hasFullNameError && (
-            <Form.Message asChild>
-              <p className="text-sm mt-1 text-alert-red">
-                Please enter your name
-              </p>
-            </Form.Message>
-          )}
-        </Form.Field>
-      )}
-      <PaymentElement
-        options={{
-          layout: {
-            type: 'accordion',
-            defaultCollapsed: false,
-            radios: false,
-            spacedAccordionItems: true,
-          },
-          readOnly,
+    <Form.Root onSubmit={submitHandler}>
+      <CheckoutCheckbox
+        isRequired={showConsentError}
+        termsOfService={cmsCommonContent.termsOfServiceUrl}
+        privacyNotice={cmsCommonContent.privacyNoticeUrl}
+        notifyCheckboxChange={(consentCheckbox) => {
+          setFormEnabled(consentCheckbox);
+          setShowConsentError(true);
         }}
       />
-      {!isPaymentElementLoading && (
-        <Form.Submit asChild>
-          <Localized id="next-new-user-submit">
+      <div
```

Comment:
This is missing the `mt-14` which adds a bit of space between the checkbox and the `Enter your card information`.